xlators/mgmt/glusterd/src/glusterd-utils.c

```diff
@@ -14884,6 +14893,44 @@ glusterd_check_brick_order(dict_t *dict, char *err_str, int32_t type,
         }
     }
 
+    /* This flag is set to test hostnames with existing bricks in the
+     * volume when when replica count is changes and brick(s) are added
+     * to same replica set.
+     */
+    if (flag) {
+        ret = glusterd_volinfo_find(*volname, &volinfo);
+        if (ret) {
+            gf_msg(this->name, GF_LOG_ERROR, 0, GD_MSG_VOL_NOT_FOUND,
+                   "Unable to find volume: %s", *volname);
+            goto out;
+        }
+        cds_list_for_each_entry(brickinfo, &volinfo->bricks, brick_list)
+        {
+            ret = getaddrinfo(brickinfo->hostname, NULL, NULL, &ai_info);
+            if (ret != 0) {
+                ret = 0;
+                gf_msg(this->name, GF_LOG_ERROR, 0,
+                       GD_MSG_HOSTNAME_RESOLVE_FAIL,
+                       "unable to resolve host name for addr %s",
+                       brickinfo->hostname);
+                goto out;
+            }
+            pre_list_tmp1 = MALLOC(sizeof(addrinfo_list_t));
+            if (pre_list_tmp1 == NULL) {
+                ret = 0;
```

Comment:
why are we assigning ret = 0, here?

Reply:
removed it. Did it while testing just to get pass through the function.